src/components/board/SharingTabSidebard.vue

```diff
@@ -3,7 +3,8 @@
 		<multiselect v-model="addAcl" :options="unallocatedSharees" label="label"
 			@input="clickAddAcl" @search-change="asyncFind">
 			<template #option="scope">
-				{{ scope.option.label }}
+				<avatar :user="scope.option.value.shareWith" />
+				<span class="avatarLabel">{{ scope.option.label }} </span>
```

Comment:
You could just use the User layout option of our multiselect component for this:
https://nextcloud-vue-components.netlify.com/#/Components?id=avatarselectoption

Then we also have nice icons to differentiate between users, groups or circles 
![image](https://user-images.githubusercontent.com/3404133/63924349-686de780-ca48-11e9-8250-d6fdcbb48b77.png)